examples/siro_sandbox/sandbox_app.py

```diff
@@ -681,15 +740,63 @@ def get_grasp_release_controls_text():
             text_delta_y=-50,
         )
 
-    def sim_update(self, dt):
-        # todo: pipe end_play somewhere
+    def record_task_state(self):
```

Comment:
This function records a few bits and pieces of task/sim state that might be helpful later. I'm somewhat speculating what will be needed later.

Reply:
For example, if we later wanted to compute "distance traveled by each agent", we could do it by inspecting the agent translations here.